### PR TITLE
Add new function

### PR DESCRIPTION
**Pull request by jiratQ**
add function change_cardsidx_to_str
and debug some .copy()

### CODE REVIEW

File: src/poker/CheckPriority.py [hunk 1]
```diff
@@ -216,7 +216,7 @@ def value(use: List[str]):
         return [1e3+twopair[0]+twopair[1]*100,'twopair',[left],use]
         #tie  pass
     elif(onepair!=0): 
-        zz=use
+        zz=use.copy()
```
Comment (namirinz) [inline src/poker/CheckPriority.py:219]: What is zz ?

Comment (jiratQ) [reply src/poker/CheckPriority.py:219]: new list that keep card in hands instead of use
because we have modified use by ,pop()

**Review by namirinz — APPROVED** [review]
Nice one!

**Review by phusitsom — APPROVED** [review]
Nice one!

**Pull request merged**